test/WebSocket.test.js

```diff
@@ -1616,6 +1616,24 @@ describe('WebSocket', function() {
       });
     });
 
+    it('should remove added event listener', function(done) {
+      this.timeout(4000);
```

Comment:
There is no need to increase the timeout please keep the default value by removing this line.